db/sqlglue.c

```diff
@@ -5415,7 +5361,7 @@ int sqlite3BtreeMovetoUnpacked(BtCursor *pCur, /* The cursor to be moved */
     if (pCur->bt->is_temporary) {
         if (pIdxKey) {
             if (bdb_is_hashtable(pCur->tmptable->tbl)) {
-                Mem mem = {0};
+                Mem mem = {{0}};
```

Comment:
What was the warning? Did you check if this really works (or only suppresses the warning)?